tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -78,10 +79,27 @@ public TensorFlowInferenceInterface(AssetManager assetManager, String model) {
         throw new RuntimeException("Failed to load model from '" + model + "'", e);
       }
     }
+    
     try {
-      loadGraph(is, g);
+      if (VERSION.SDK_INT >= 18) {
+        Trace.beginSection("initializeTensorFlow");
+        Trace.beginSection("readGraphDef");
+      }
+    
+      // TODO(ashankar): Can we somehow mmap the contents instead of copying them?
+      byte[] graphDef = new byte[is.available()];
+
```

Comment:
I think we forgot to actually read the bytes into the `graphDef` array  here.
i.e., an:

`is.read(graphDef)`

is required?

Reply:
Sent a fix in #13073